fontforge/sfd.c

```diff
@@ -4855,8 +4857,7 @@ void SFDGetKerns( FILE *sfd, SplineChar *sc, char* ttok ) {
 		while ( (ch=nlgetc(sfd))==' ' );
 		ungetc(ch,sfd);
 		if ( ch=='{' ) {
-		    kp->adjust=chunkalloc(sizeof(DeviceTable));
-		    SFDReadDeviceTable(sfd,kp->adjust);
+		    kp->adjust = SFDReadDeviceTable(sfd, NULL);
```

Comment:
Okay, but it was already being done like this in the past. If you have a separate allocation as before, there's a possibility that you will have allocated memory but `SFDReadDeviceTable` will return NULL due to some error.